spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala

```diff
@@ -203,6 +212,13 @@ object WriteConf {
 
     val ttlOption = if(ttlSeconds == TTLParam.default) TTLOption.defaultValue else TTLOption.constant(ttlSeconds)
```

Comment:
Would you mind fixing the formatting here? I know it's not your patch but I see it now and it burns :)